app/assets/stylesheets/styles/dashboard.scss

```diff
@@ -0,0 +1,22 @@
+// Place all the styles related to the dashboard controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+nav .nav-wrapper {
+  padding-right: 30px;
+  i {
+    font-size: 1.2rem;
+  }
+}
+
+.user_info {
+  background-color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.